BUILD.bazel

```diff
@@ -34,7 +34,7 @@ go_binary(
     pure = "on",
     static = "on",
     visibility = ["//visibility:public"],
-    x_defs = {"github.com/buchgr/bazel-remote/server.GitCommit": "{STABLE_GIT_COMMIT}"},
```

Comment:
why rename this?

Reply:
I moved the variable to the main package because that's where it's first used, and it seems mostly unrelated to the server package. I'm not sure how to refer to the main package in the "import" naming style, but this seems to work.

The rules_go documentation seems to suggest creating a version library, would you prefer that?
https://github.com/bazelbuild/rules_go/blob/master/go/core.rst#defines-and-stamping